source/ed/build_helper.cpp

```diff
@@ -250,14 +254,15 @@ VJ& VJ::operator()(const std::string& sp_name,
         sp->idx = b.data->pt_data->stop_points.size();
         sp->name = sp_name;
         sp->uri = sp_name;
-        if (!b.data->pt_data->networks.empty())
+        if (!b.data->pt_data->networks.empty()) {
             sp->network = b.data->pt_data->networks.front();
+        }
 
         b.sps[sp_name] = sp;
         b.data->pt_data->stop_points.push_back(sp);
         auto sa_it = b.sas.find(sp_name);
         if (sa_it == b.sas.end()) {
-            navitia::type::StopArea* sa = new navitia::type::StopArea();
+            auto* sa = new navitia::type::StopArea();
```

Comment:
Can I have no raw pointers in the code base for :christmas_tree: :santa: :christmas_tree: ?

Reply:
I have this dream too but it won't be as easy as that... Plus I am not sure if the effort is worth the benefit...